fluid/PaddleCV/int8/utility.py

```diff
@@ -0,0 +1,336 @@
+import paddle.fluid.core as core
```

Comment:
core is not part of public API, avoid using it.

Reply:
Do we have an alternative one?